### PR TITLE
Pass next/image as unoptimized

### PR DESCRIPTION
**Pull request by zvonimirr**
This will make the linter happy and allow for loading of favicons without adding the domain to the whitelist

### CODE REVIEW

File: components/Header.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ export function Header() {
 
   return (
     <header>
-      <Link href="/">
```
Comment (strdr4605) [inline components/Header.tsx:36]: I don't think this is needed here as the `h1` does not have a `href` prop
image

File: pages/index.tsx [hunk 2]
```diff
@@ -141,12 +142,12 @@ export default function Home() {
...
                   <>
                     {feed.favicon && (
-                      // eslint-disable-next-line @next/next/no-img-element
-                      <img
+                      <Image
                         alt={feed.title}
                         src={feed.favicon}
                         width={16}
                         height={16}
+                        unoptimized
```
Comment (strdr4605) [inline pages/index.tsx:150]: Thanks, did not know about this option.

File: components/Header.tsx [hunk 3]
```diff
@@ -33,7 +33,7 @@ export function Header() {
...
+      <Link href="/" passHref>
```
Comment (strdr4605) [inline components/Header.tsx:36]: I don't think this is needed here as the `h1` does not have a `href` prop
image

**Review by strdr4605 — CHANGES_REQUESTED** [review]

File: components/Header.tsx [hunk 4]
```diff
@@ -33,10 +33,12 @@ export function Header() {
 
   return (
     <header>
-      <Link href="/">
-        <h1>
-          <span>C</span>lient-<span>o</span>nly <span>Feed</span> Reader
-        </h1>
+      <Link href="/" passHref>
+        <a>
```
[the remark this answers is not in the document]
Comment (zvonimirr) [reply components/Header.tsx:37]: Got it 😀

**Review by strdr4605 — APPROVED** [review]

**Pull request merged**